src/main/java/games/strategy/ui/DoubleTextField.java

```diff
@@ -2,21 +2,20 @@
 
 import java.awt.event.FocusAdapter;
 import java.awt.event.FocusEvent;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
 
 import javax.swing.JTextField;
 import javax.swing.text.AttributeSet;
 import javax.swing.text.BadLocationException;
 import javax.swing.text.PlainDocument;
 
-import games.strategy.util.ListenerList;
-
 public class DoubleTextField extends JTextField {
   private static final long serialVersionUID = -8698753345852617343L;
   private double m_max = Double.MAX_VALUE;
   private double m_min = Double.MIN_VALUE;
   private String m_terr = null;
-  private final ListenerList<DoubleTextFieldChangeListener> m_listeners =
-      new ListenerList<>();
+  private final List<DoubleTextFieldChangeListener> m_listeners = new CopyOnWriteArrayList<>();
```

Comment:
I would argue that a thread-safe list (`CopyOnWriteArrayList`) is not needed for a UI component, which is single threaded.  But, as it's been that way for 10+ years, changing it now is outside the scope of this PR. :smile:  However, this change should help us better spot stuff like that in the future.